serverFiles/app.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const getAllTopics = require('./controllers/topics.controllers');
+
+const app = express();
+
+app.get('/', (req, res, next) => {
+  res.status(404).send({ msg: 'Not found' });
+});
+
+app.get('/api/topics', getAllTopics);
+
+app.use((err, req, res, next) => {
```

Comment:
I can't see a test which triggers a 404 in this block.  What is the motivation for this error handling middleware function? Did you mean it to be a 404?